eos/b-decays/b-to-d-l-nu.cc

```diff
@@ -126,12 +150,29 @@ namespace eos
         return integrate<GSL::QNG>(f, s_min, s_max);
     }
 
+    // normalized_differential_branching_ratio (|V_cb|=1)
+    double
+    BToDLeptonNeutrino::normalized_differential_branching_ratio(const double & s) const
+    {
+        return _imp->normalized_differential_branching_ratio(s);
+    }
+    
+    // normalized(|Vcb|=1) integrated branching_ratio including NP
+    double
+    BToDLeptonNeutrino::normalized_integrated_branching_ratio(const double & s_min, const double & s_max) const
+    {
+        std::function<double (const double &)> f = std::bind(&Implementation<BToDLeptonNeutrino>::normalized_differential_branching_ratio,
+                                                             _imp.get(), std::placeholders::_1);
+        
+        return integrate<GSL::QNG>(f, s_min, s_max);
```

Comment:
Does it work with QNG? Or is QAGS a better choice?

Reply:
Yes, in my case I tried this and QNG works when integrating over the entire q2 phase space. I don't know whether QAGS is a better choice though.

Reply:
Ok, if QNG works then we should keep it. It's generally faster than QAGS.